state-chain/pallets/cf-swapping/src/tests.rs

```diff
@@ -2128,3 +2140,22 @@ fn deposit_address_ready_event_contain_correct_boost_fee_value() {
 		);
 	});
 }
+
+#[test]
+fn can_update_multiple_items_at_once() {
```

Comment:
nit: we're only updating one at once here now

Reply:
It's updating the max swap amounts for btc and dot - that's two?